js/state/frameStateUtil.js

```diff
@@ -233,27 +213,49 @@ function getPartition (frameOpts) {
   }
   return partition
 }
+
+function cloneFrame (frameOpts, guestInstanceId) {
```

Comment:
I think this actually needs to selectively include properties instead of exclude them. Unlike a webview that is detached and reattached, a cloned webview won't have the same audio playback for instance